arbnode/execution/executionengine.go

```diff
@@ -599,7 +599,7 @@ func (s *ExecutionEngine) Start(ctx_in context.Context) {
 			s.latestBlockMutex.Lock()
 			block := s.latestBlock
 			s.latestBlockMutex.Unlock()
-			if block != lastBlock && block != nil {
+			if block != nil && (lastBlock == nil || block.TxHash() != lastBlock.TxHash()) {
```

Comment:
I think this should be block.Hash and lastBlock.Hash not TxHash

Reply:
Done.